docs/conf.py

```diff
@@ -12,10 +12,10 @@
 import os
 import sys
 
+import pandablocks
```

Comment:
This needed to be after the insert on sys.path so that sphinx-build could find the module. If moving it earlier works then we can remove the sys.path.insert below. If that works, please can you update the template module with this change

Reply:
As of https://github.com/PandABlocks/PandABlocks-client/pull/24/commits/53ec0027e20db4e9eef8b05e8088165e79c66488 I think it seems to be safe to remove the path mangling code. I can build the docs on both my machine and in CI.